superset-frontend/src/views/CRUD/dashboard/DashboardList.tsx

```diff
@@ -106,6 +110,61 @@ class DashboardList extends React.PureComponent<Props, State> {
   initialSort = [{ id: 'changed_on_delta_humanized', desc: true }];
 
   columns = [
+    {
+      Cell: ({ row: { original } }: any) => {
+        // TODO: return null if no user is logged in
+        const fetchFaveStar = (id: number) => {
+          SupersetClient.get({
+            endpoint: `${FAVESTAR_BASE_URL}/${id}/count/`,
+          })
+            .then(({ json }) => {
+              const faves = {
+                ...this.state.favoriteStatus,
+              };
+
+              faves[id] = json.count > 0;
+
+              this.setState({
+                favoriteStatus: faves,
+              });
+            })
+            .catch(() => {});
```

Comment:
let's handle this exception. Swallowed exceptions make debugging very difficult. There should be other examples of handling promise errors in this component.

Reply:
What action should we be taking on an error? Should we be displaying a toast or just quietly logging the error?

Reply:
Toasts would be good, at the very least we should be giving a reason for favorite action failing as that could cause some confusion for the user.